client/src/styles/_modal.less

```diff
@@ -24,7 +24,7 @@
   left: 0;
   right: 0;
 
-  width: 636px;
+  width: 638px;
```

Comment:
I like that change. :mag:

Reply:
Due to bordering this was the only way to give exact 636 pixels of width to modals actually. With 636 pixels assigned, the modals would have 634 pixels of width. Took me a while to fix this.

Reply:
<img width="843" alt="Screenshot 2020-03-06 at 10 51 19" src="https://user-images.githubusercontent.com/15003836/76072540-71f87f00-5f98-11ea-9338-a1c48520f59a.png">

Reply:
I personally think we should bot be picky about two pixels. 

Otherwise we'd need to specify what the width is about => Includes border? Does not include border?

Reply:
To me it's no big deal neither. It's just that the design review includes the exact with of the modal as well as the exact width of the input fields, that's why I modified here, to make sure the PR passes the design review.

Reply:
What was specified? Should the width include the border or not?

Reply:
I'm  not exactly sure, this is how it looks like in the Zeplin:
<img width="774" alt="Screenshot 2020-03-06 at 11 45 30" src="https://user-images.githubusercontent.com/15003836/76076954-20ec8900-5fa0-11ea-8ae3-6105471f239b.png">

@nazlikaya Could you clarify this conversation?

Reply:
Yea. Just checked, Zeplin, too. I guess it shows the outer width of elements (including potential border).

@nazlikaya So given we had a 5px border, an element in Zeplin would still have the same width it had with a 1px border?

Reply:
The number Zeplin shows the total width and height of the element (including borders).

Reply:
Ok, I take this modification back.

Reply:
Update: Nazli says we should have exactly 636 px's visible in Chrome Dev Tool so I'm keeping this field as 638px. Had to update Checkbox and Radio widths as well.